app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -10,16 +10,19 @@ import java.io.ByteArrayOutputStream
 import java.io.IOException
 import java.net.MalformedURLException
 import java.net.URL
+import kotlin.math.roundToInt
 
-class TransparentTileOWM(private val tileType: String):TileProvider {
+class TransparentTileOWM(private val tileType: String) : TileProvider {
     private val opacityPaint = Paint()
 
     /**
      * Sets the desired opacity of map [Tile]s, as a percentage where 0% is invisible and 100% is completely opaque.
      * @param opacity The desired opacity of map [Tile]s (as a percentage between 0 and 100, inclusive)
      */
-    fun setOpacity(opacity: Int) {
-        val alpha = Math.round(opacity * 2.55).toInt() // 2.55 = 255 * 0.01
+    //Only used here so set to private.
+    private fun setOpacity(opacity: Int) {
+        //Changed to kotlin function .roundToInt instead of .round and .toInt see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/round-to-int.html
```

Comment:
Ditto previous comment.

Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.